Classes/KIFUITestActor.h

```diff
@@ -76,7 +76,7 @@ static inline KIFDisplacement KIFDisplacementForSwipingInDirection(KIFSwipeDirec
 /*!
  @abstract Waits until a view or accessibility element is present.
  @discussion The view or accessibility element with the given label is found in the view hierarchy. If the element isn't found, then the step will attempt to wait until it is. Note that the view does not necessarily have to be visible on the screen, and may be behind another view or offscreen. Views with their hidden property set to YES are ignored.
- 
```

Comment:
Your editor changed the whitespace of a bunch of lines. Please get rid of these changes so we get a clear diff for history.